build.gradle

```diff
@@ -67,6 +67,7 @@ subprojects {
         it.options.fork = true
         it.options.compilerArgs += ["-Xlint:unchecked", "-Xlint:deprecation", "-Xlint:-options"]
         it.options.encoding = "UTF-8"
+        it.options.compilerArgs << "-Werror"
```

Comment:
Can we add `"-Werror"` in `it.options.compilerArgs +=` line?

Reply:
This worked
`it.options.compilerArgs += ["-Xlint:unchecked", "-Xlint:deprecation", "-Xlint:-options", "-Werror"]`